canonical-websites/www.ubuntu.com
# 1612 /download/cloud/autopilot to vbt

## Done

* update /download/cloud/autopilot to vbt

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/download/cloud/autopilot)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/9.%20Download?preview=download-cloud-autopilot.png) and the [demo](http://www.ubuntu.com-1612-cloud-openstack-autopilot.demo.haus/download/cloud/autopilot)

## Issue / Card

Fixes #1612
